Setup/Product.wxs

```diff
@@ -135,6 +138,312 @@
         <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Runtime.InteropServices.RuntimeInformation.dll" />
       </Component>
 
+            <Component Id="Microsoft.Win32.Primitives.dll" Guid="*">
```

Comment:
❗️ I'm not convinced all of these are required. I'd like to review the method in which they were determined in order to verify.

Reply:
.net standard is already confusing enough.  ALL I am focused on is getting no warnings and keeping dependencies consistent.  By all means create an issue to use correct .net standard.

Reply:
Pointers are welcome.  This .net standard blaa = .net framework x is not very easy to work with.  Package refs help and figure it out for you which is what I'm depending on.  Sure would be nice if they suggested a different package if we mixed different .net standard versions.

Reply:
The compiled assemblies might only reference a subset of the assemblies in the NuGet packages. You can use System.Reflection.Metadata to build a transitive closure of the true dependencies. One example can be found in https://github.com/tunnelvisionlabs/vsxdeps/tree/master/DependencyAnalyzer/DependencyAnalyzer, but keep in mind that project only looks at the dependencies of the public API surface, where you'll want all the dependencies declared by the assemblies.

Reply:
Here is the premise I am going by:

If it is in release folder after a rebuild, it was placed there for a reason.  If it is a dll or a exe then it should be included.  When I include the files and run in portable or installed, I do not get any issues of can't load blaaaa.  I frankly don't care if it needs a million files or not.  I wrote a script and hashed files in the release folder with paths to find unique files.  I then parsed the installer script for files.  If the path was not found in the installer then I generated the right component xml.

Reply:
Build log.  https://drive.google.com/file/d/1k1K__oDpvkQYWjRdKt9IaHgW_fiBznU8/view?usp=sharing

Reply:
If you are so unsure, you can

1. Run ```git clean -d -x -f```
1. Make sure .\GitExtensions\bin\release is empty
1. Run .\Setup\Prepare-Release.ps1 -oldVersion 3.0.0 -newVersion 3.0.1 -milestones '49'
1. Look at .\GitExtensions\bin\release for yourself.  

Build log. https://drive.google.com/file/d/1k1K__oDpvkQYWjRdKt9IaHgW_fiBznU8/view?usp=sharing